packages/buttons/stories/9-ToggleIcon.stories.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { ToggleIconButton } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import Icon from '@zendeskgarden/svg-icons/src/16/eye-stroke.svg';
+
+export default {
+  title: 'Components/Buttons/ToggleIconButton',
+  component: ToggleIconButton
+} as Meta;
+
+export const Default: Story = ({
+  isPrimary,
+  isDanger,
+  isBasic,
+  isPill,
+  focusInset,
+  isRotated,
+  isPressed,
+  disabled,
+  size
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <ToggleIconButton
+          aria-label="See more"
+          isPrimary={isPrimary}
+          isDanger={isDanger}
+          isBasic={isBasic}
+          isPill={isPill}
+          focusInset={focusInset}
+          isRotated={isRotated}
```

Comment:
[nit] it might be good to select an icon that can indicate `isRotated` is working as expected.